app/controllers/users_controller.rb

```diff
@@ -12,13 +13,14 @@ class UsersController < ApplicationController
   def index
     @user = current_user
     @gds = current_user.permissions.admin_management
-
     if @gds
       if params['team_id']
         @team = Team.find_by_id(params['team_id'])
         @team_members = get_users_in_group(group_name: @team.team_alias).map { |cognito_user| as_team_member(cognito_user: cognito_user) }
       else
         @teams = Team.all
+        @rps, @idps, @other = @teams.group_by(&:team_type).values_at(TEAMS::RP, TEAMS::IDP, TEAMS::OTHER)
+        @all_rps_and_idps = @teams - @other unless @other.nil?
```

Comment:
```suggestion
```
Think these can come out?

Reply:
`@all_rps_and_idps = @teams - @other unless @other.nil?` can be removed. The other 3 are used in the view to print all the teams out